src/components/Contact/Contact.css

```diff
@@ -0,0 +1,52 @@
+.contact {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+  min-height: 300px;
+  flex-direction:column;
+}
+
+.contact_bg{
+  background-image: url('../../../static/images/2.jpg');
+  width: 100%;
+  height: auto;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: right;
+  border-radius: 5px;
+}
+
+.contact_bg-overlay{
+  width: 100%;
+  height: auto;
+  background: rgba(40, 167, 69, 0.6);
+  color: #fff;
+  text-align: center;
+  padding-top: 2rem;
+}
+
+.contact_bg-overlay p{
+  font-size: 20px;
+}
+
+.contact_bg .btn{
+  font-size: 18px;
+  color:rgba(40, 167, 69);
+}
+
+section p{
+  font-size:15px;
+}
+
+.container{
+color: #606060;
+font-size: 12px;
+line-height: 1.5em;
+outline: 0;
+font-family: "Nunito Sans",sans-serif!important;
+box-sizing: border-box;
+zoom: 1;
+padding: 3rem 0;
+margin: 0 auto;
+text-align: center; }
```

Comment:
Why are you overriding `.container` class here? This results breaking the theme of the whole website.
- Too much top and bottom padding.
- Things are centred which shouldn't be.
- Unnecessary CSS rulesets like `zoom`, `outline`.
- Also there is no new/empty line at the end of the file.
The above are just some of the initial observations.

![image](https://user-images.githubusercontent.com/23737560/77231845-77261280-6bc3-11ea-9b02-90556449f272.png)